pkg/github/github.go

```diff
@@ -93,13 +117,118 @@ func NewClient(ctx context.Context, token string, opts ...Option) *GitHubClient
 	return g
 }
 
+// ListIssues lists all issues and returns their numbers in a repository matching the given criteria.
+func (g *GitHubClient) ListIssues(ctx context.Context, owner, repo string, labels []string, state string) ([]*Issue, error) {
```

Comment:
This is a generic function, so it would probably be better to accept a `github.issueListByRepoOptions` instead of labels. It's not immediately clear that this filters by labels (nor is it reusable for other queries).

Reply:
Done